### ISSUE TITLE
Has this registry been deployed

### ISSUE THREAD

**Issue by tomerds** [manifoldxyz/royalty-registry-solidity#1]
Hi, 

I am just looking in to royalties for an NFT project and was wondering if you have deployed this registry ?

Thanks!

**Comment by wwhchung** [comment]
In the process of getting buy-in from a few of the larger Marketplaces and figuring out governance.

**Comment by tomerds** [comment]
Cool thanks for letting me know :) 

I was having a look at your [FVCK_CRYSTAL](https://etherscan.io/address/0x7afeda4c714e1c0a2a1248332c100924506ac8e6#code) contract as example, am I reading correctly that you implement an interface for each marketplaces (Rarible v1/v2 and Foundations) approach to royalties? Does this mean if you sold a FVCK_CRYSTAL token on one of these platforms it would process the royalties correctly out the box?

Awesome approach if so!

**Comment by wwhchung** [comment]
Yes, this is correct.  Rarible v2 marketplace accepts v1 as well, so we've implemented the v1 spec as well as EIP 2981.

**Comment by dievardump** [comment]
I've seen the contract, but I had no ideas that Foundation allows sales from contracts that are not theirs.

Is there a doc or anything about this somewhere?

**Comment by tomerds** [comment]
From the look of their contract it seems possible to sell any NFT through the protocol, but their [FAQs](https://help.foundation.app/en/articles/5046913-how-do-i-connect-my-opensea-account-to-foundation) imply that the listings in the UI are restricted to only FND minted tokens.

**Comment by wwhchung** [comment]
> I've seen the contracts implementing foundations royalties, but I had no ideas that Foundation allows sales from contracts that are not theirs.
> 
> Is there a doc or anything about this somewhere?

They just started integrating external contracts. 

https://twitter.com/withFND/status/1443300628810272770
